src/core/lib/surface/server.h

```diff
@@ -385,9 +383,9 @@ class Server : public ServerInterface,
   };
 
   struct Listener {
-    explicit Listener(OrphanablePtr<ListenerInterface> l)
+    explicit Listener(RefCountedPtr<ListenerInterface> l)
         : listener(std::move(l)) {}
-    OrphanablePtr<ListenerInterface> listener;
+    RefCountedPtr<ListenerInterface> listener;
```

Comment:
This should stay an `OrphanablrPtr`.  The intended usage of `InternallyRefCounted` is that the external owner uses `OrphanablePtr`, so that `Orphan()` triggers shutdown and eventually gives up the internal refs.  Only the internal refs are `RefCountedPtr`.